modules/core/src/precomp.hpp

```diff
@@ -55,6 +55,7 @@
 #include "opencv2/core/cuda.hpp"
 #include "opencv2/core/opengl.hpp"
 #include "opencv2/core/va_intel.hpp"
+#include "opencv2/core/va_intel_interop.hpp"
```

Comment:
We don't need both of them for the whole core module. It is better to remove it from here.

Especially if they are included below through 2 `#if` guards.

Reply:
I understand

Reply:
Done b56bc32